lib/media_web/router.ex

```diff
@@ -53,9 +53,6 @@ defmodule Media.Routes do
 
         get("/", PageController, :index, as: :media)
         # left here for reference
-        #  on how to upload media from a form to S3
-        post("/upload", PageController, :upload, as: :media)
```

Comment:
why are you removing this route ?

Reply:
this route was there for testing purposes @jfayad

Reply:
so I guess the comment at line 55 should be removed too, no ?

Reply:
removed